client/signup/wpcom-login-form/test/index.jsx

```diff
@@ -12,103 +14,98 @@ describe( 'WpcomLoginForm', () => {
 	};
 
 	test( 'should render default fields as expected.', () => {
-		const wrapper = shallow( <WpcomLoginForm { ...props } />, { disableLifecycleMethods: true } );
+		const { container, rerender } = render( <WpcomLoginForm { ...props } /> );
 
 		// should render root form element
-		const form = wrapper.find( 'form' );
-		expect( form ).toHaveLength( 1 );
-		expect( form.prop( 'method' ) ).toEqual( 'post' );
-		expect( form.prop( 'action' ) ).toEqual( 'https://test.wordpress.com/wp-login.php' );
+		const form = container.firstChild;
+		expect( form ).toBeVisible();
+		expect( form ).toHaveAttribute( 'method', 'post' );
+		expect( form ).toHaveAttribute( 'action', 'https://test.wordpress.com/wp-login.php' );
 
 		// form should include default hidden elements
-		expect( wrapper.find( 'form > input[type="hidden"]' ) ).toHaveLength( 4 );
-		expect( wrapper.find( 'form > input[name="log"]' ).prop( 'value' ) ).toEqual( 'log_text' );
-		expect( wrapper.find( 'form > input[name="pwd"]' ).prop( 'value' ) ).toEqual( 'secret' );
-		expect( wrapper.find( 'form > input[name="authorization"]' ).prop( 'value' ) ).toEqual(
+		expect( container.querySelector( 'input[name="log"]' ) ).toHaveValue( 'log_text' );
+		expect( container.querySelector( 'input[name="pwd"]' ) ).toHaveValue( 'secret' );
+		expect( container.querySelector( 'input[name="authorization"]' ) ).toHaveValue(
 			'authorization_token'
 		);
-		expect( wrapper.find( 'form > input[name="redirect_to"]' ).prop( 'value' ) ).toEqual(
+		expect( container.querySelector( 'input[name="redirect_to"]' ) ).toHaveValue(
 			'https://test.wordpress.com'
 		);
 
 		// when update a prop
-		wrapper.setProps( { log: 'another_log' } );
-		expect( wrapper.find( 'form > input[name="log"]' ).prop( 'value' ) ).toEqual( 'another_log' );
+		rerender( <WpcomLoginForm { ...props } log="another_log" /> );
+		const el = container.querySelector( 'input[name="log"]' );
+		expect( el.value ).toEqual( 'another_log' );
```

Comment:
Can you use `.toHaveValue` for this check, too?

Reply:
Good point. Fixed via https://github.com/Automattic/wp-calypso/pull/68099/commits/ad0818b88280c63514203362ae13584cf8c2b510 👍🏻